standalone/src/models/commons/base-money/base-money-draft/fields-config.ts

```diff
@@ -0,0 +1,11 @@
+import { type TModelFieldsConfig } from '@/core';
+import type { TBaseMoneyDraftGraphql } from '../types';
+
+// This model only exists in the GraphQL API
```

Comment:
According to the GraphQL API, the `PriceDraft` entity `value` field has this type:
```ts
export type TCtpBaseMoneyInput = {
  centPrecision?: InputMaybe<TCtpMoneyInput>;
  highPrecision?: InputMaybe<TCtpHighPrecisionMoneyInput>;
};
```

It does not seem to have something similar for the REST version as the `value` field is defined like this for that one:
```ts
export type _Money = Money | TypedMoney | TypedMoneyDraft;
```